charts/prometheus-kafka-exporter/README.md

```diff
@@ -4,29 +4,27 @@ A Prometheus exporter for [Apacher Kafka](https://kafka.apache.org/) metrics.
 
 This chart bootstraps a [Kafka Exporter](https://github.com/danielqsj/kafka_exporter) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
 
-
 ## Prerequisites
 
-- Kubernetes 1.8+ with Beta APIs enabled
+- Kubernetes 1.19+
+- Helm 3
+
+Helm v2 was no longer supported from chart version 2.0.0.
 
-## Get Repo Info
+## Get repository Info
```

Comment:
Will you please, change "repository" to "Repository" as is the case in other headings?

Reply:
fixed.  https://github.com/prometheus-community/helm-charts/pull/3199/commits/e5c13e9aa397ea2d97025c0748d978d24cc26117